### ISSUE TITLE
Incorrect heading order with service accordions on health services pages

### ISSUE THREAD

**Issue by laflannery** [department-of-veterans-affairs/va.gov-cms#12082]
## Describe the defect
On VAMC health services pages, the accordions titles all have an H3 heading, which is correct. However, this means that the headings within the accordions should be H4s instead of H3s.

## To Reproduce
1. Go to either VAMC health services page, like [VA Boston](https://www.va.gov/boston-health-care/health-services/)
2. Scroll down to the service accordions section
3. Expand one of the accordions
4. Use the dev tools inspector, or a headings browser plugin to identify the heading levels of the headings within

## Screenshots
![Screenshot 2024-07-17 at 9 44 12 AM](https://github.com/user-attachments/assets/e3535a91-60c0-4e50-bc5a-3ec9ea7cc846)

## AC / Expected behavior

- [x] Confirm if template change will impact other stuff. Say where if yes. **Confirmed that this template change only affects VAMC Health Services pages.**
- [ ] Update headings within the accordions to be H4s
- [ ] Accessibility review

**Comment by davidmpickett** [comment]
Remove Lovell label as this is a problem on all VAMCs

**Comment by briandeconinck** [comment]
This issue was identified during the Collaboration Cycle Staging Review for Lovell VAMC on 2/28/2023. Applicable Collaboration Cycle labels have been added to this ticket for tracking and dashboard purposes, rather than creating a duplicate ticket.

**Comment by kmariepat-cityfriends** [comment]
This is going to be solved by another organization issue (how do we get rid of full rich text editors) moving to the low priority backlog

**Comment by davidmpickett** [comment]
Semi-related quandary: https://github.com/department-of-veterans-affairs/va.gov-cms/issues/13143

**Comment by CherylEvans** [comment]
This ticket was recreated in va.gov-team Ticket #[64839](https://app.zenhub.com/workspaces/collaboration-cycle-reviews-61d89e9e5f4c120011a0ad37/issues/gh/department-of-veterans-affairs/va.gov-team/64839)

**Comment by davidmpickett** [comment]
I think this is the template: https://github.com/department-of-veterans-affairs/content-build/blob/main/src/site/facilities/health_service.drupal.liquid

**Comment by randimays** [comment]
Pulled in for Sprint 20.
